src/api/grainConfig.js

```diff
@@ -4,19 +4,24 @@ import {type DistributionPolicy} from "../core/ledger/applyDistributions";
 import * as G from "../core/ledger/grain";
 import * as C from "../util/combo";
 import * as NullUtil from "../util/null";
+import {toDiscount} from "../core/ledger/grainAllocation";
 
 export type GrainConfig = {|
   +immediatePerWeek: number,
   +balancedPerWeek: number,
+  +recentPerWeek: number,
+  +recentWeeklyDecayRate?: number,
```

Comment:
Add a comment explaining what this parameter will do (e.g. will choosing 0.1 result in high or low decay? If I want "total decay" i.e. the immediate policy, would that correspond to 0 or to 1)?

Reply:
I added more detail in `src/core/ledger/grainAllocation.js` where the Recent policy is typed.  I think it would also be worthwhile to add details to the template instance, but I feel like the confusion around naming here is indicative of a larger problem of readability.  I plan on changing the grain config process to be clearer semantically in regards to typing.